frontend/src/features/properties/map/activity/detail/ActivityContainer.tsx

```diff
@@ -126,18 +123,9 @@ export const getActivityPropertiesOnFile = (
   activityFileProperties: Api_PropertyActivity[],
 ): Api_PropertyFile[] => {
   const activityFilePropertyIds = activityFileProperties.map(af => af.propertyFileId);
-
-  if (file?.fileType === FileTypes.Research) {
-    return (file as Api_ResearchFile).researchProperties
-      ?.filter(researchProperty => activityFilePropertyIds.includes(researchProperty.id))
-      .map(p => ({ id: p.id })) as Api_PropertyFile[];
-  } else if (file?.fileType === FileTypes.Acquisition) {
-    return (file as Api_AcquisitionFile).acquisitionProperties
-      ?.filter(acquisitionProperty => activityFilePropertyIds.includes(acquisitionProperty.id))
-      .map(p => ({ id: p.id })) as Api_PropertyFile[];
-  } else {
-    throw Error('Unexpected file type');
-  }
+  return file.fileProperties
```

Comment:
creating this common fileProperties logic allows this function to be simplified.